src/lib/commandBarConfigs/settingsCommandConfig.ts

```diff
@@ -129,10 +125,16 @@ export function createSettingsCommand({
     icon: 'settings',
     needsReview: false,
     onSubmit: (data) => {
-      if (data !== undefined && data !== null) {
-        send({ type: `set.${type}`, data })
+      if (
+        data !== undefined &&
+        data !== null &&
+        'value' in data &&
+        'level' in data
+      ) {
+        const coercedData = data as unknown as SetEventTypes['data']
+        actor.send({ type: `set.${type}`, data: coercedData })
       } else {
-        send({ type })
+        console.error('Invalid data submitted to settings command')
```

Comment:
Would be nice to have the data output? Makes debugging quick / could also show up in the coredump

Reply:
On it